pgaudit-11.yaml

```diff
@@ -0,0 +1,38 @@
+# pgAudit v1.3.X is intended to support PostgreSQL 11.
+# https://github.com/pgaudit/pgaudit/blob/master/README.md#postgresql-version-compatibility
+package:
+  name: pgaudit-11
```

Comment:
this could get confusing having the package name as pgaudit-11 whenever the version of pgaudit is pgaudit-1.3.

That said if we change it, it'll be as equally confusing when the version doesn't match the postgres version. ACK their docs for version compatibilty: https://github.com/pgaudit/pgaudit/blob/master/README.md#postgresql-version-compatibility.

All that considered i think this is fine as is